.github/workflows/windows.yml

```diff
@@ -404,6 +404,13 @@ jobs:
           type: 'Debug'
           tflags: '~1516'
           config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DCURL_USE_LIBPSL=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DCURL_USE_WOLFSSL=ON -DCURL_USE_GSASL=ON'
+        - name: 'gnutls'
+          install: 'brotli zlib zstd libpsl nghttp2 shiftmedia-libgnutls libssh2 pkgconf gsasl'
```

Comment:
FTR there is no pristine GnuTLS for MSVC. There is the fork from ShiftMedia and its vcpkg port. Quality unknown.

Reply:
I think the problem is coming from ngtcp2. when I adding only shiftmedia-libgnutls it not create this.

Reply:
Also because I adding the shiftmedia-libgnutls to curl tests, we can estimate the quality of this port.

Reply:
`ngtcp2`'s crypto lib links to the Release GnuTLS DLL instead of the Debug one. Isn't this closer to the root cause?

Reply:
OK, answer here: https://github.com/curl/curl/pull/14495#issuecomment-2284542160

Reply:
This thread wasn't about the root cause.